### DISCUSSION TITLE
关于3in1空调新风地暖温控器实体的建议

### DISCUSSION THREAD

**Discussion by vanscer** [XiaoMi/ha_xiaomi_home#269] [Ideas]
空调、新风、地暖3in1温控器，接入后控制都没有问题，但是实体是switch、number、select的形式，然后就只有switch、select能接入HomeKit（select接入后也是switch形式），能否将空调实体修改为climate、新风修改为fan
![image](https://github.com/user-attachments/assets/a9ad49f7-75b9-431e-a921-ba355828e333)

**Comment by SusanPhevos** [comment]
同 #209

**Reply by vanscer** [reply]
了解，期待后期更新～

**Comment by SusanPhevos** [comment]
请帮忙验证一下 #470  功能是否正常

**Reply by vanscer** [reply]
![image](https://github.com/user-attachments/assets/02ce83ba-a6ac-4924-a58d-72b46f3dcf71)
climate实体是有了，但是，首先climate实体上面有个空调开/关的switch，需要先打开switch才可以在climate中调节空调模式（如果不打开switch，在climate中调节了任一模式，都只是在控制器上跳转了一下界面并没有实际打开），其次，climate中调节温度的控制逻辑是紊乱的，在climate中的任何空调模式下调节温度，调节的都是地暖温度。
![image](https://github.com/user-attachments/assets/4beabfe6-e52d-465b-8710-7adfd5e2b310)
地暖控制部分都变得不可用。
![image](https://github.com/user-attachments/assets/a23c2205-1467-4210-a8ab-6c531fa3f32a)
传感器部分，温湿度传感器变得不可用。
